### PR TITLE
feat: rating

### PR DESCRIPTION
**Pull request by munjalSaksham71**
Added Rating Component for E-commerce.

### CODE REVIEW

**Review by Manju-sree — COMMENTED** [review]
code looks fine, but please add screen shots for preview

**Comment by Manju-sree** [comment]
netlify preview don't have rating feature in it

**Comment by munjalSaksham71** [comment]
> netlify preview don't have rating feature in it

Rating feature is at very bottom.

**Comment by munjalSaksham71** [comment]
Screenshot 2022-02-09 at 12 46 10 PM

**Comment by munjalSaksham71** [comment]
Screenshot 2022-02-09 at 4 24 50 PM

File: docs/docs.html [hunk 1]
```diff
@@ -382,6 +382,26 @@ <h1 class="heading">REGAL UI</h1>
...
+            <div class="overview">Ratings provide insight regarding others' opinions and experiences, and can allow the user to submit a rating of their own.</div>
+            <div class="heading2 mt-1 mb-1">User Rating</div> <br>
+            <div class="wrap mb-3">
+                <span class="fa fa-star checked"></span>
+                <span class="fa fa-star checked"></span>
+                <span class="fa fa-star checked"></span>
+                <span class="fa fa-star "></span>
+                <span class="fa fa-star"></span>
+            </div>
+            <p class="headin4">3 average based on 300 reviews.</p> <br>
```
Comment (paarthbajaj) [inline docs/docs.html]: Is the spelling correct here? "headin". I think it is a type.

**Review by paarthbajaj — APPROVED** [review]
Seems good, just check my comment once. You can merge then.

File: docs/docs.html (hunk 1 above)
Comment (munjalSaksham71) [reply docs/docs.html]: yeah, its typo - Thanks!!

**Pull request merged**